src/easydmp/dmpt/models.py

```diff
@@ -2146,6 +2147,58 @@ def validate_choice(self, data):
         return False
 
 
+class StorageForecastQuestion(NoCheckMixin, Question):
+    """A non-branch-capable question for RDA DMP Common Standard Cost
+
+    Only title is required.
+
+    The framing text for the canned answer utilizes the Django template system,
+    not standard python string formatting. If there is no framing text
+    a serialized version of the raw choice is returned.
+    """
+
+    DEFAULT_FRAMING_TEXT = """<dl class="answer-cost">{% for obj in choices %}
+<dt>{{ obj.year }}
+<span>{{ obj.storage_estimate }} TiB</span>
+</dt>
+{% endfor %}
+</dl>
+"""
+
+    class Meta:
+        proxy = True
+
+    def save(self, *args, **kwargs):
+        self.input_type = 'storageforecast'
+        super().save(*args, **kwargs)
+
+    def get_canned_answer(self, choice, **kwargs):
+        if not choice:
+            return self.get_optional_canned_answer()
+
+        framing_text = self.framing_text if self.framing_text else self.DEFAULT_FRAMING_TEXT
```

Comment:
self.framing_text or self.DEFAULT_FRAMING_TEXT?

Reply:
self.framing_text is an optional field on the model IIRC. `DEFAULT_FRAMING_TEXT` needs to be set to something sensible regardless.

Reply:
I meant can't this be a little bit shorter as `framing_text = self.framing_text or self.DEFAULT_FRAMING_TEXT` ? :)

Reply:
I've been trying to train myself to use the "offical" trinary operator, which is `x if y else z`, but, hm..